Ark.Tools.ResourceWatcher/ResourceWatcher.cs

```diff
@@ -415,6 +418,89 @@ public class ProcessContext
         public ResultType? ResultType { get; set; }
         public int? Index { get; set; }
         public int? Total { get; set; }
+
+        public bool IsNewResource(out Tuple<string,LocalDateTime?,LocalDateTime?> changed)

```

Comment:
'New' is misleading. This is more IsResourceUpdated() 

New means Last==null (not present before).